ERROR - 2025-10-12 11:48:26 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\WebSystemITE-G9\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\WebSystemITE-G9\app\Controllers\Dashboard.php(52): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(933): App\Controllers\Dashboard->index()
#7 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#8 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\WebSystemITE-G9\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
ERROR - 2025-10-12 11:48:26 --> Error loading dashboard data: Unknown column 'date' in 'where clause'
CRITICAL - 2025-10-12 11:51:07 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: appointments/update/2]
in SYSTEMPATH\BaseModel.php on line 1823.
 1 SYSTEMPATH\BaseModel.php(1823): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(994): CodeIgniter\BaseModel->transformDataToArray([], 'update')
 3 SYSTEMPATH\Model.php(860): CodeIgniter\BaseModel->update([...], [])
 4 APPPATH\Controllers\Appointment.php(331): CodeIgniter\Model->update('2', [])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Appointment->update('2')
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Appointment))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-10-12 11:52:02 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\WebSystemITE-G9\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\WebSystemITE-G9\app\Controllers\Dashboard.php(52): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(933): App\Controllers\Dashboard->index()
#7 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#8 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\WebSystemITE-G9\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
ERROR - 2025-10-12 11:52:02 --> Error loading dashboard data: Unknown column 'date' in 'where clause'
ERROR - 2025-10-12 12:03:06 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\WebSystemITE-G9\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\WebSystemITE-G9\app\Controllers\Dashboard.php(52): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(933): App\Controllers\Dashboard->index()
#7 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#8 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\WebSystemITE-G9\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
ERROR - 2025-10-12 12:03:06 --> Error loading dashboard data: Unknown column 'date' in 'where clause'
ERROR - 2025-10-12 12:03:08 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\WebSystemITE-G9\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\WebSystemITE-G9\app\Controllers\Dashboard.php(52): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(933): App\Controllers\Dashboard->index()
#7 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#8 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\WebSystemITE-G9\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
ERROR - 2025-10-12 12:03:08 --> Error loading dashboard data: Unknown column 'date' in 'where clause'
ERROR - 2025-10-12 13:38:51 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\WebSystemITE-G9\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\WebSystemITE-G9\app\Controllers\Dashboard.php(52): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(933): App\Controllers\Dashboard->index()
#7 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#8 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\WebSystemITE-G9\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
ERROR - 2025-10-12 13:38:51 --> Error loading dashboard data: Unknown column 'date' in 'where clause'
ERROR - 2025-10-12 13:42:12 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\WebSystemITE-G9\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\WebSystemITE-G9\app\Controllers\Dashboard.php(52): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(933): App\Controllers\Dashboard->index()
#7 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#8 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\WebSystemITE-G9\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
ERROR - 2025-10-12 13:42:12 --> Error loading dashboard data: Unknown column 'date' in 'where clause'
ERROR - 2025-10-12 14:13:53 --> mysqli_sql_exception: Duplicate entry '' for key 'patient_id' in C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `pa...', 0)
#1 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pa...')
#2 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pa...')
#3 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pa...', Array, false)
#4 C:\xampp\htdocs\WebSystemITE-G9\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\WebSystemITE-G9\system\BaseModel.php(838): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\WebSystemITE-G9\system\Model.php(797): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\WebSystemITE-G9\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\WebSystemITE-G9\app\Controllers\Admin\Patients.php(77): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(933): App\Controllers\Admin\Patients->processRegister()
#10 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Patients))
#11 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\WebSystemITE-G9\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-10-12 14:13:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'patient_id'
[Method: POST, Route: admin/patients/register]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (:first_name:, :last_name:, :email:, :phone:, :gender:, :date_of_birth:, :address:, :blood_type:, :emergency_contact:, :medical_history:, :status:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Admin\Patients.php(77): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\Patients->processRegister()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Patients))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-12 14:13:53 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'patient_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (\'Jerald jhon\', \'Maca\', \'geraldmacafernan5dez@gmail.com\', \'099 781 5058\', \'male\', \'2025-10-12\', \'BULA, PHASE 2 BLOCK 9 LOT 42 GENSANVILLE SUB\', \'A\', NULL, \'\', \'active\', \'2025-10-12 14:13:53\', \'2025-10-12 14:13:53\')')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (\'Jerald jhon\', \'Maca\', \'geraldmacafernan5dez@gmail.com\', \'099 781 5058\', \'male\', \'2025-10-12\', \'BULA, PHASE 2 BLOCK 9 LOT 42 GENSANVILLE SUB\', \'A\', NULL, \'\', \'active\', \'2025-10-12 14:13:53\', \'2025-10-12 14:13:53\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (:first_name:, :last_name:, :email:, :phone:, :gender:, :date_of_birth:, :address:, :blood_type:, :emergency_contact:, :medical_history:, :status:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Admin\Patients.php(77): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\Patients->processRegister()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Patients))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-12 14:13:53 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'patient_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (\'Jerald jhon\', \'Maca\', \'geraldmacafernan5dez@gmail.com\', \'099 781 5058\', \'male\', \'2025-10-12\', \'BULA, PHASE 2 BLOCK 9 LOT 42 GENSANVILLE SUB\', \'A\', NULL, \'\', \'active\', \'2025-10-12 14:13:53\', \'2025-10-12 14:13:53\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (\'Jerald jhon\', \'Maca\', \'geraldmacafernan5dez@gmail.com\', \'099 781 5058\', \'male\', \'2025-10-12\', \'BULA, PHASE 2 BLOCK 9 LOT 42 GENSANVILLE SUB\', \'A\', NULL, \'\', \'active\', \'2025-10-12 14:13:53\', \'2025-10-12 14:13:53\')')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (\'Jerald jhon\', \'Maca\', \'geraldmacafernan5dez@gmail.com\', \'099 781 5058\', \'male\', \'2025-10-12\', \'BULA, PHASE 2 BLOCK 9 LOT 42 GENSANVILLE SUB\', \'A\', NULL, \'\', \'active\', \'2025-10-12 14:13:53\', \'2025-10-12 14:13:53\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (:first_name:, :last_name:, :email:, :phone:, :gender:, :date_of_birth:, :address:, :blood_type:, :emergency_contact:, :medical_history:, :status:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Admin\Patients.php(77): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\Patients->processRegister()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Patients))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-10-12 14:14:02 --> mysqli_sql_exception: Duplicate entry '' for key 'patient_id' in C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `pa...', 0)
#1 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pa...')
#2 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pa...')
#3 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pa...', Array, false)
#4 C:\xampp\htdocs\WebSystemITE-G9\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\WebSystemITE-G9\system\BaseModel.php(838): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\WebSystemITE-G9\system\Model.php(797): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\WebSystemITE-G9\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\WebSystemITE-G9\app\Controllers\Admin\Patients.php(77): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(933): App\Controllers\Admin\Patients->processRegister()
#10 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Patients))
#11 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\WebSystemITE-G9\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-10-12 14:14:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'patient_id'
[Method: POST, Route: admin/patients/register]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (:first_name:, :last_name:, :email:, :phone:, :gender:, :date_of_birth:, :address:, :blood_type:, :emergency_contact:, :medical_history:, :status:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Admin\Patients.php(77): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\Patients->processRegister()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Patients))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-12 14:14:02 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'patient_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (\'Jerald jhonhg\', \'Macagfg\', \'geraldmacafernan5dez@gmail.com\', \'099 781 5058\', \'male\', \'2025-10-12\', \'BULA, PHASE 2 BLOCK 9 LOT 42 GENSANVILLE SUB\', \'A\', NULL, \'\', \'active\', \'2025-10-12 14:14:02\', \'2025-10-12 14:14:02\')')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (\'Jerald jhonhg\', \'Macagfg\', \'geraldmacafernan5dez@gmail.com\', \'099 781 5058\', \'male\', \'2025-10-12\', \'BULA, PHASE 2 BLOCK 9 LOT 42 GENSANVILLE SUB\', \'A\', NULL, \'\', \'active\', \'2025-10-12 14:14:02\', \'2025-10-12 14:14:02\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (:first_name:, :last_name:, :email:, :phone:, :gender:, :date_of_birth:, :address:, :blood_type:, :emergency_contact:, :medical_history:, :status:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Admin\Patients.php(77): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\Patients->processRegister()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Patients))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-12 14:14:02 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'patient_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (\'Jerald jhonhg\', \'Macagfg\', \'geraldmacafernan5dez@gmail.com\', \'099 781 5058\', \'male\', \'2025-10-12\', \'BULA, PHASE 2 BLOCK 9 LOT 42 GENSANVILLE SUB\', \'A\', NULL, \'\', \'active\', \'2025-10-12 14:14:02\', \'2025-10-12 14:14:02\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (\'Jerald jhonhg\', \'Macagfg\', \'geraldmacafernan5dez@gmail.com\', \'099 781 5058\', \'male\', \'2025-10-12\', \'BULA, PHASE 2 BLOCK 9 LOT 42 GENSANVILLE SUB\', \'A\', NULL, \'\', \'active\', \'2025-10-12 14:14:02\', \'2025-10-12 14:14:02\')')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (\'Jerald jhonhg\', \'Macagfg\', \'geraldmacafernan5dez@gmail.com\', \'099 781 5058\', \'male\', \'2025-10-12\', \'BULA, PHASE 2 BLOCK 9 LOT 42 GENSANVILLE SUB\', \'A\', NULL, \'\', \'active\', \'2025-10-12 14:14:02\', \'2025-10-12 14:14:02\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (:first_name:, :last_name:, :email:, :phone:, :gender:, :date_of_birth:, :address:, :blood_type:, :emergency_contact:, :medical_history:, :status:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Admin\Patients.php(77): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\Patients->processRegister()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Patients))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-10-12 14:16:47 --> mysqli_sql_exception: Duplicate entry '' for key 'patient_id' in C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `pa...', 0)
#1 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pa...')
#2 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pa...')
#3 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pa...', Array, false)
#4 C:\xampp\htdocs\WebSystemITE-G9\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\WebSystemITE-G9\system\BaseModel.php(838): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\WebSystemITE-G9\system\Model.php(797): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\WebSystemITE-G9\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\WebSystemITE-G9\app\Controllers\Admin\Patients.php(77): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(933): App\Controllers\Admin\Patients->processRegister()
#10 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Patients))
#11 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\WebSystemITE-G9\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-10-12 14:16:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'patient_id'
[Method: POST, Route: admin/patients/register]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (:first_name:, :last_name:, :email:, :phone:, :gender:, :date_of_birth:, :address:, :blood_type:, :emergency_contact:, :medical_history:, :status:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Admin\Patients.php(77): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\Patients->processRegister()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Patients))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-12 14:16:47 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'patient_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (\'Jerald jhonhg\', \'Macagfg\', \'geraldmacafernan5dez@gmail.com\', \'099 781 5058\', \'male\', \'2025-10-12\', \'BULA, PHAuSE 2 BLOCK 9 LOT 42 GENSANVILLE SUB\', \'A\', NULL, \'\', \'active\', \'2025-10-12 14:16:47\', \'2025-10-12 14:16:47\')')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (\'Jerald jhonhg\', \'Macagfg\', \'geraldmacafernan5dez@gmail.com\', \'099 781 5058\', \'male\', \'2025-10-12\', \'BULA, PHAuSE 2 BLOCK 9 LOT 42 GENSANVILLE SUB\', \'A\', NULL, \'\', \'active\', \'2025-10-12 14:16:47\', \'2025-10-12 14:16:47\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (:first_name:, :last_name:, :email:, :phone:, :gender:, :date_of_birth:, :address:, :blood_type:, :emergency_contact:, :medical_history:, :status:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Admin\Patients.php(77): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\Patients->processRegister()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Patients))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-12 14:16:47 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'patient_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (\'Jerald jhonhg\', \'Macagfg\', \'geraldmacafernan5dez@gmail.com\', \'099 781 5058\', \'male\', \'2025-10-12\', \'BULA, PHAuSE 2 BLOCK 9 LOT 42 GENSANVILLE SUB\', \'A\', NULL, \'\', \'active\', \'2025-10-12 14:16:47\', \'2025-10-12 14:16:47\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (\'Jerald jhonhg\', \'Macagfg\', \'geraldmacafernan5dez@gmail.com\', \'099 781 5058\', \'male\', \'2025-10-12\', \'BULA, PHAuSE 2 BLOCK 9 LOT 42 GENSANVILLE SUB\', \'A\', NULL, \'\', \'active\', \'2025-10-12 14:16:47\', \'2025-10-12 14:16:47\')')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (\'Jerald jhonhg\', \'Macagfg\', \'geraldmacafernan5dez@gmail.com\', \'099 781 5058\', \'male\', \'2025-10-12\', \'BULA, PHAuSE 2 BLOCK 9 LOT 42 GENSANVILLE SUB\', \'A\', NULL, \'\', \'active\', \'2025-10-12 14:16:47\', \'2025-10-12 14:16:47\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `patients` (`first_name`, `last_name`, `email`, `phone`, `gender`, `date_of_birth`, `address`, `blood_type`, `emergency_contact`, `medical_history`, `status`, `created_at`, `updated_at`) VALUES (:first_name:, :last_name:, :email:, :phone:, :gender:, :date_of_birth:, :address:, :blood_type:, :emergency_contact:, :medical_history:, :status:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Admin\Patients.php(77): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin\Patients->processRegister()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Patients))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-10-12 14:34:00 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\WebSystemITE-G9\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\WebSystemITE-G9\app\Controllers\Dashboard.php(52): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(933): App\Controllers\Dashboard->index()
#7 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#8 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\WebSystemITE-G9\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
ERROR - 2025-10-12 14:34:00 --> Error loading dashboard data: Unknown column 'date' in 'where clause'
ERROR - 2025-10-12 14:53:05 --> mysqli_sql_exception: Unknown column 'date' in 'where clause' in C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\WebSystemITE-G9\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\WebSystemITE-G9\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\WebSystemITE-G9\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\WebSystemITE-G9\app\Controllers\Dashboard.php(52): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(933): App\Controllers\Dashboard->index()
#7 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#8 C:\xampp\htdocs\WebSystemITE-G9\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\WebSystemITE-G9\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\WebSystemITE-G9\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
ERROR - 2025-10-12 14:53:05 --> Error loading dashboard data: Unknown column 'date' in 'where clause'
CRITICAL - 2025-10-12 14:55:29 --> ErrorException: Cannot redeclare App\Models\PatientModel::generatePatientId()
[Method: GET, Route: patients/view]
in APPPATH\Models\PatientModel.php on line 76.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
